packages/synapse-react-client/src/components/widgets/Range.tsx

```diff
@@ -13,13 +13,11 @@ export type RangeProps = {
   initialValues?: RangeValues
   className?: string
   errorText?: string
-  onChange: (newValues: RangeValues) => void
+  onApplyClicked: (newValues: RangeValues) => void
```

Comment:
`onApplyClicked` is more clear since this component contains two input fields that the user can update. Hopefully it is now obvious that this callback only is invoked when the user clicks "Apply"